hr_attendance_report_theoretical_time/security/hr_attendance_report_theoretical_time_security.xml

```diff
@@ -2,6 +2,11 @@
 <!-- Copyright 2018 Tecnativa - Pedro M. Baeza
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo noupdate="1">
+    <record id="rule_multi_company_theoretical_vs_worked_report" model="ir.rule">
+        <field name="name">Theoretical vs worked hours multi-company</field>
+        <field name="model_id" ref="model_hr_attendance_theoretical_time_report" />
+        <field name="domain_force">[('company_id', 'in', company_ids + [False])]</field>
```

Comment:
`company_id` of an employee can be empty?

Reply:
Right, it is a required field, it has been changed.